src/test/java/com/softserveinc/ita/mkaralash/RozetkaTest.java

```diff
@@ -0,0 +1,40 @@
+package com.softserveinc.ita.mkaralash;
+
+import com.softserveinc.ita.pageobjects.mkaralash.GoogleHomePage;
+import com.softserveinc.ita.pageobjects.mkaralash.GoogleSearchResultPage;
+import com.softserveinc.ita.pageobjects.mkaralash.RozetkaHomePage;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class RozetkaTest extends TestRuner {
+
+    @Test
+    public void rozetkaTest() {
+        GoogleHomePage googleHomePage = new GoogleHomePage(driver);
```

Comment:
in methods, you could use local variable 'var' instead of strict variable type

`var googleHomePage = new GoogleHomePage(driver);`